src/style.css

```diff
@@ -1 +1,95 @@
 /* add styles here */
+:root {
+  --border-sol-col: 1px solid #e1e7ed;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.form, .form__search {
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.form__search {
+  height: 100%;
+  position: relative;
+}
+
+.img__search{
```

Comment:
The 'background-image' property should be used along with 'background-repeat', 'background-position' and 'background-size' properties for a better control on the image.